packages/medusa-plugin-brightpearl/src/services/brightpearl.js

```diff
@@ -111,7 +108,7 @@ class BrightpearlService extends BaseService {
         httpMethod: "POST",
         uriTemplate: `${this.options.backend_url}/brightpearl/goods-out`,
         bodyTemplate:
-          "{\"account\": \"${account-code}\", \"lifecycle_event\": \"${lifecycle-event}\", \"resource_type\": \"${resource-type}\", \"id\": \"${resource-id}\" }",
+          '{"account": "${account-code}", "lifecycle_event": "${lifecycle-event}", "resource_type": "${resource-type}", "id": "${resource-id}" }',
```

Comment:
todo: We don't know the repercussions of changing this, so I would keep it out of this PR.

Reply:
I think this is my linter that has done that. But the resulted string will be exactly the same